frontend/packages/erd-core/src/features/erd/components/ERDRenderer/CommandPalette/CommandPalette.tsx

```diff
@@ -14,6 +14,13 @@ import { TableNode } from '../../ERDContent/components'
 import styles from './CommandPalette.module.css'
 import { useCommandPalette } from './CommandPaletteProvider'
 
+const getTableLinkHref = (activeTableName: string) => {
+  if (typeof window === 'undefined') return
```

Comment:
This line is to avoid a ReferenceError during SSR.
In practice, no special workaround is needed to avoid hydration errors, since the `getTableLinkHref` function is only called for link elements that are rendered on the client (because the dialog is guaranteed to be closed during SSR)

However, I admit it looks a bit strange 😅 
I think we should separate the dialog wrapper and the dialog content into different files to simplify the implementation.
The dialog content is guaranteed to be rendered only on the client.